File: online-feedback-collection-system-backend/src/main/java/com/dyashin/onlinefeedbackcollectionsystem/service/UserService.java
	package com.dyashin.onlinefeedbackcollectionsystem.service;
	
	import com.dyashin.onlinefeedbackcollectionsystem.dto.UserDTO;
	import com.dyashin.onlinefeedbackcollectionsystem.entity.User;
	
	public interface UserService {
	
		boolean register(UserDTO userDTO);
	
		User login(String userEmail, String userPassword);
	
	}

File: online-feedback-collection-system-frontend/src/service/userService.js
import axios from "axios";

const API_BASE_URL = "http://localhost:5080/user";

export const registerUser = (userData) => {
  return axios.post(`${API_BASE_URL}/register`, userData);
};


export const loginUser = async (userEmail, userPassword) => {
  try {
    const res = await axios.post("http://localhost:5080/user/login", {
      userEmail,
      userPassword
    });
    return res.data.record;
  } catch (err) {
    throw err.response?.data || { message: "Login failed" };
  }
};
